Sources/SwiftToolchain/Manifest.swift

```diff
@@ -122,31 +94,29 @@ struct DumpedManifest: Decodable {
       [SwiftLanguageVersion]?.self,
       forKey: .swiftLanguageVersions
     )
-    // TODO: Change to `PackageDependencyDescription` when we move to `Swift 5.4`
     let dependencies = try container.decode(
-      [DumpedPackageDependencyDescription].self,
+      [PackageDependencyDescription].self,
       forKey: .dependencies
     )
     let products = try container.decode([ProductDescription].self, forKey: .products)
     let targets = try container.decode([TargetDescription].self, forKey: .targets)
     let platforms = try container.decode([PlatformDescription].self, forKey: .platforms)
-    // TODO: Change to non-optional when we move to `Swift 5.4`
-    // `packageKind` is not available in dumped JSON with pre-5.4 compilers.
-    let packageKind = try? container.decode(PackageReference.Kind.self, forKey: .packageKind)
```

Comment:
Can carton still parse 5.4 package JSON?

Reply:
I haven't tested it, but I'm fine with dropping support for SwiftWasm 5.4 here if that's the easiest way to get 5.5 out.